sys/conf/ldscript.arm64

```diff
@@ -105,15 +106,16 @@ SECTIONS
   .sdata     : { *(.sdata) }
   _edata  =  .;
   PROVIDE (edata = .);
-  __bss_start = .;
+  . = ALIGN(32);
```

Comment:
Why 32?

Reply:
I think I mistakenly assumed that we needed to align to twice the data size for `stp` as it is used to clear.

Reply:
Yeah 16 is sufficient for that